docs/source/overview/cli.md

```diff
@@ -0,0 +1,499 @@
+
+#### BED file sampler
+The BED file sampler loads a dataset from a `.bed` file. This can be generated by one of the online samplers in Selene with the `save_dataset` parameter. 
+
+An example configuration for a BED file sampler:
+```YAML
+sampler: !obj:selene_sdk.samplers.file_samplers.BedFileSampler {
+    filepath: /path/to/data.bed,
+    reference_sequence: !obj:selene_sdk.sequences.Genome {
+        input_path: /path/to/reference_sequence.fa
+    },
+    n_samples: 640000,
+    sequence_length: 1000,
+    targets_avail: True,
+    n_features: 919,
+}
+```
+
+##### Parameters
+- `filepath`: Path to the BED file.
+- `reference_sequence`: Path to a reference sequence FASTA file we can query to create our data samples.
+- `n_samples`: Number of lines in the file. (`wc -l <filepath>`)
+- `sequence_length`: Optional, default is None. If the coordinates of each sample in the BED file, already account for the full sequence (that is, the columns `end - start = sequence_length`, there is no need to specify this parameter. If `sequence_length` is not None, the length of each sample will be checked to determine whether the sample coordinates need to be adjusted to match the sequence length expected by the model architecture.
+- `targets_avail`: Optional, default is False. If `targets_avail`, assumes that it is the last column of the `.bed` file. The last column should contain the indices, separated by semicolons, of features (classes) found within a given sample's coordinates (e.g. 0;1;45;60). This format assumes that we are only looking for the absence/presence of each feature within the interval.
+- `n_features`: Optional, default is None. If `targets_avail` is True, must specify `n_features`, the total number of features (classes).
+
+#### Matrix file sampler
+The matrix file sampler loads a dataset from a matrix file.
+
+An example configuration for a matrix file sampler:
+```YAML
+sampler: !obj:selene_sdk.samplers.file_samplers.MatFileSampler {
+    filepath: /path/to/data.mat,
+    sequence_key: sequences,
+    targets_key: targets,
+    random_seed: 123,
+    shuffle: True,
+    sequence_batch_axis: 0,
+    sequence_alphabet_axis: 1,
+    targets_batch_axis: 0
+}
+```
+
+##### Parameters
+- `filepath`: The path to the file from which to load the data. 
+- `sequence_key`: The key for the sequences data matrix.
+- `targets_key`: Optional, default is `None`. The key to the targets data matrix.
+- `random_seed`: Optional, default is 436. Sets the random seed for sampling.
+- `shuffle`: Optional, default is `True`. Shuffle the order of the samples in the matrix before sampling from it.
+- `sequence_batch_axis`: Optional, default is 0. Specify the batch axis for the sequences matrix.
+- `sequence_alphabet_axis`: Optional, default is 1. Specify the alphabet axis.
+- `targets_batch_axis`: Optional, default is 0. Specify the batch axis for the targets matrix.
```

Comment:
At the very end, as a reader, I would like to see an expanded YAML file with all those parts that were just explained being used. I know you have a link in the beginning, but having it here would make it easier to comprehend as a whole